src/pages/popover/popover.ts

```diff
@@ -1,18 +1,27 @@
+import { LoginPage } from './../login/login';
 import { SettingsPage } from './../settings/settings';
 import { Component } from '@angular/core';
 import { NavController, NavParams, ViewController } from 'ionic-angular';
+import { SettingService } from './../../providers/setting-service';
 
 @Component({
   selector: 'page-popover',
   templateUrl: 'popover.html'
 })
 export class PopoverPage {
 
-  constructor(public navCtrl: NavController, public navParams: NavParams, public viewCtrl: ViewController ) { }
+  constructor(public navCtrl: NavController, public navParams: NavParams, public viewCtrl: ViewController, public settingService: SettingService) { }
 
 
   loadSettings() {
     this.navCtrl.push(SettingsPage);
     this.viewCtrl.dismiss();
   }
+
+  logoutUser() {
+    this.settingService.clearLoginData().subscribe();
```

Comment:
I think this is wrong. It has nothing to do with the persistent settings when we log out a user. What we need to do is clearing the credentials in the AuthService